README.rst

```diff
@@ -137,11 +137,26 @@ dirs::
     '/Users/trentm/Library/Logs/SuperApp/1.0'
 
 
-Contributing and About
-======================
+Why the Fork?
+=============
 
 This repository is a friendly fork of the wonderful work started by
 `ActiveState <https://github.com/ActiveState/appdirs>`_ who created
 ``appdirs``, this package's ancestor.
 
-Contributions are most welcome.
+Maintaining an open source project is no easy task, particularly
+from within an organization, and the Python community is indebted
+to ``appdirs`` (and to Trent Mick and Jeff Rouse in particular) for
+creating an incredibly useful simple module, as evidenced by the wide
+number of users it has attracted over the years.
+
+Nonetheless, given the number of long-standing open issues
+and pull requests, and no clear path towards `ensuring
+that maintenance of the package would continue or grow
+<https://github.com/ActiveState/appdirs/issues/79>`_, this fork was
+created.
+
+Should the upstream package become active again, it is entirely within the
+realm of possibility that we combine efforts.
```

Comment:
I'd prefer we not say/do this, as we are better equipped to provide long-term maintenance by virtue of having write access and us using this in other projects we directly own (https://github.com/pypa/virtualenv/pull/2142, https://github.com/DataDog/integrations-core/pull/9356, etc.).

Also, combining efforts would require them to incorporate breaking changes we'll make soon like dropping old Python versions and possibly returning `Path` objects. At that point, users might as well use this new library 🙂

Reply:
Fair! My intention by saying this was not the "our changes get merged back to theirs and we go away" model, more as you say "in such a hypothetical, we continue to maintain and *add* them, because we believe the package needs more attention" -- but happy to remove (or clarify further) if that's still better left unsaid. Just say the word.

Reply:
I'm +0 on removing as it adds ambiguity regarding whether or not the user's dependency choice of the 2 will point to the hypothetic combined effort one.

Reply:
Fair. Done!